src/test/features/response/helpers/taskListBuilder.ts

```diff
@@ -34,5 +36,54 @@ describe('TaskListBuilder', () => {
         expect(taskList.tasks.map(task => task.name)).to.not.contain('Free mediation')
       })
     })
+
+    describe('Is Response Partially Rejected Due To task', () => {
+      beforeEach(() => {
+        stub = sinon.stub(ResponseDraft.prototype, 'isResponsePartiallyRejectedDueTo')
+      })
+
+      it('should be enabled when, when will you pay is available', () => {
+        stub.withArgs(RejectPartOfClaimOption.AMOUNT_TOO_HIGH).returns(true)
+
+        const input = {
+          howMuchOwed: {
+            amount: 200,
+            text: 'I owe nothing'
+          }
+        }
+        const responseDraft: ResponseDraft = new ResponseDraft().deserialize(input)
+        const taskList: TaskList = TaskListBuilder.buildRespondToClaimSection(responseDraft, moment(), undefined)
+        expect(taskList.tasks.map(task => task.name)).to.contain('When will you pay?')
+      })
+
+      it('should be disabled when, when will you pay is not available', () => {
+        stub.returns(false)
+
+        const taskList: TaskList = TaskListBuilder.buildRespondToClaimSection(new ResponseDraft(), moment(), undefined)
+        expect(taskList.tasks.map(task => task.name)).to.not.contain('When will you pay?')
+      })
+
+      it('should be enabled when how much paid the claimant is available', () => {
```

Comment:
This and next cases should be under its own `describe` section. Each task list entry should be separate scenario with its own `describe` section

Reply:
Ok